### PR TITLE
remove references to git.openssl.org from REEADME.md

### LINKED ISSUE #25191: git.openssl.org seems to be down.
I have been using the documented git.openssl.org method for cloning openssl into a project.
This seems to have stopped working.

The README.md still has it listed as an option:
https://github.com/openssl/openssl?tab=readme-ov-file#for-testing-and-development

> A local copy of the Git Repository can be obtained by cloning it from the original OpenSSL repository using
> 
> ```
> git clone git://git.openssl.org/openssl.git
> ```

However the connection times out:

```
$ git ls-remote git://git.openssl.org/openssl.git HEAD     
fatal: unable to connect to git.openssl.org:
git.openssl.org[0: 142.250.204.19]: errno=Operation timed out
git.openssl.org[1: 2404:6800:4006:814::2013]: errno=Operation timed out
```

The alternative GitHub link is still working:

```
$ git ls-remote https://github.com/openssl/openssl.git HEAD
ca112fccdd34a8538f14ddf8c3569b8331eae357	HEAD
```

Do you know if the git.openssl.org link is just temporary down, or is the recommendation to switch to the GitHub link from now on?

### PR DESCRIPTION
**Pull request by Sashan**
Fixes #25191

### CODE REVIEW

**Review by levitte — DISMISSED** [review]

**Review by t8m — APPROVED** [review]

**Review by andrewkdinh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -86,16 +86,8 @@ the source tarballs, having a local copy of the git repository with
...
-There is a GitHub mirror of the repository at [github.com/openssl/openssl],
-which is updated automatically from the former on every commit.
-
 A local copy of the Git Repository can be obtained by cloning it from
-the original OpenSSL repository using
-
-    git clone git://git.openssl.org/openssl.git
-
-or from the GitHub mirror using
+from the GitHub
```
Comment (FdaSilvaYY) [inline README.md:90]: Typo in commit message.
From is duplicated with the From on the end of previous line.

the GitHub **repository**.

**Review by shahsb — APPROVED** [review]

**Review by tom-cosgrove-arm — CHANGES_REQUESTED** [review]
Before this gets merged, can we fix [the issues](https://github.com/openssl/openssl/pull/25198/files#r1718630404) @FdaSilvaYY points out?

a/REEADME.md/README.md/ in the commit message

Fix the repeated word "from" at the end of (new) line 89 and the beginning of line 90

**Comment by t8m** [comment]
ping @Sashan for fixing this

File: README.md (hunk 1 above)
Comment (t8m) [reply README.md:90]: @Sashan this still needs fixing

Comment (t8m) [reply README.md:90]:
```suggestion
from the GitHub repository using
```

**Comment by nhorman** [comment]
@Sashan can you please finish this up?  It would be nice to get this closed out, and its so close

File: README.md (hunk 1 above)
Comment (tom-cosgrove-arm) [reply README.md:90]: The commit message typo and the duplicated `from` have been fixed, but `cloning it from the GitHub` isn't right: Github is a proper noun, or can be used as an adjective, so this should be changed, either just to

```
from GitHub:
```

or using Tomáš' suggestion

```
from the GitHub repository using
```

**Pull request closed without merging**